requestor-tutorials/service-development/service-example-2-managed-erigon.md

```diff
@@ -38,7 +38,127 @@ A typical interaction flow proceeds in the following manner:
 
 ## Runtime implementation
 
+Here we demonstrate how self-contained Exe-Unit runtimes can be implemented, tested and finally plugged into the Provider's daemon.
+
+:warning: Please be aware that custom Exe-Unit runtime doesn't provide the level of isolation comparable with gvmi image. It's just executable running with the same privileges that daemon does.
+
+### Erigon runtime
+
+First, lets discuss what is the purpose of the Erigon runtime to gain some high level overview.
+
+* accepts the Ethereum network's name from the requestor agent,
+* starts Erigon binaries to sync with the chosen network and serve Erigon RPC endpoint,
+* generates credentials for basic authentication which is implemented with nginx
+* credentials are returned to the requestor agent so user of the service can connect to the Erigon RPC.
+
+Please refer to the "Provider's node" in the above Architecture overview diagram.
+
+### Implementation
```

Comment:
This section needs snippets of actual code with description. See how e.g. [simple-service](https://handbook.golem.network/requestor-tutorials/service-development/service-example-1-simple-service) is written.